docs/usage/how-to/cli-mode.mdx

```diff
@@ -134,6 +134,10 @@ project details and structure. Use this when onboarding the agent to a new codeb
 You can pause the agent while it is running by pressing `Ctrl-P`. To continue the conversation after pausing, simply
 type `/resume` at the prompt.
 
+#### Using MCP servers
+
+To configure Model Context Protocol (MCP) servers, you can follow the documentation on [MCP servers](../mcp) and add them under the `[mcp]` section in the global `~/.openhands/config.toml` file.
```

Comment:
Can you include an example config file that includes both STDIO, SSE, and SHTTP example?